monocle/worker.py

```diff
@@ -897,6 +921,32 @@ def smart_throttle(self, requests=1):
         except (TypeError, KeyError):
             return False
 
+    async def add_lure_pokestop(self, pokestop):
```

Comment:
please remove this, we don't want it - this will only make us a target for niantic and please remove it (push -f) from your own repository